packages/flutter/test/widgets/stack_test.dart

```diff
@@ -799,4 +799,22 @@ void main() {
       "'alignment', or an explicit 'textDirection', to the Stack.",
     );
   });
+
+  testWidgets('Can update clipBehavior of IndexedStack',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(IndexedStack(textDirection: TextDirection.ltr));
+    final RenderIndexedStack renderObject =
+        tester.allRenderObjects.whereType<RenderIndexedStack>().first;
+    expect(renderObject.clipBehavior, equals(Clip.hardEdge));
+
+    /// Update clipBehavior to Clip.antiAlias
+
+    await tester.pumpWidget(IndexedStack(
+      textDirection: TextDirection.ltr,
+      clipBehavior: Clip.antiAlias,
+    ));
+    final RenderIndexedStack renderIndexedObject =
+      tester.allRenderObjects.whereType<RenderIndexedStack>().first;
+    expect(renderIndexedObject.clipBehavior, equals(Clip.antiAlias));
```

Comment:
@xu-baolin The test is failing when updating clipBehavior. Are my changes correct?

Reply:
Likely because `updateRenderObject` isn't updating the clip behavior (see my other comment above).

Reply:
Yes thanks that worked.